assets/js/src/core/modules/asset/hooks/use-asset-draft.ts

```diff
@@ -29,7 +29,7 @@ export const useAssetDraft = (id: number): UseAssetDraftReturn => {
 
   useEffect(() => {
     if (data !== undefined) {
-      dispatch(assetReceived({ id, ...data }))
+      dispatch(assetReceived({ ...data, id }))
```

Comment:
@vin0401 do we even need the `id`? Isn't it included in `data`?

Reply:
it should be included in the data, but due to the structure of the asset-draft-slice the id is still needed.